extensions/ql-vscode/src/view/model-editor/ModelEditor.tsx

```diff
@@ -27,14 +27,18 @@ const LoadingContainer = styled.div`
   font-weight: 600;
 `;
 
-const ModelEditorContainer = styled.div`
-  margin-top: 1rem;
-`;
+const ModelEditorContainer = styled.div``;
 
 const HeaderContainer = styled.div`
   display: flex;
   flex-direction: row;
   align-items: end;
+  background-color: var(--vscode-editorGutter-background);
```

Comment:
Is there a specific reason why this is `editorGutter-background` rather than `editor-background`? Usually these two colors match, but if we want to match the background color of the webview, we should use `editor-background`.

Reply:
Oh, just a mistake when copying the variable. Updated it!